Replica-Manager/model/RegisteredReplica.java

```diff
@@ -0,0 +1,9 @@
+package model;
+
+public enum RegisteredReplica {
+	ReplicaGab,
+	ReplicaChrist,
```

Comment:
LOL

Reply:
Ahah we can change it later for maybe DEMS1, DEMS2...

Reply:
I know im that great, just glad u mention it =p 


![christ](http://www.jesuschristsavior.net/Savior.jpeg)